torch/csrc/jit/runtime/static/impl.cpp

```diff
@@ -121,6 +121,7 @@ bool mayContainAlias(
   }
   return db.mayContainAlias(as, bs);
 }
+}
```

Comment:
Where does the extra right bracket come from?

Reply:
opening and closing the anonymous namespace - i did this so that you could more easily see the diff. after you approve i will move these two functions lower out of the anonymous namespace.